phpunit.xml

```diff
@@ -15,8 +15,8 @@
 	<logging>
         <log type="tap" target="build/report.tap"/>
         <log type="junit" target="build/report.junit.xml"/>
+        <log type="coverage-text" target="php://stdout"/>
         <log type="coverage-html" target="build/coverage" charset="UTF-8" yui="true" highlight="true"/>
-        <log type="coverage-text" target="build/coverage.txt"/>
```

Comment:
I never noticed that. Not something I did, and I doubt there's a good reason to keep it in a file. :-)